cleanlab/models/fasttext.py

```diff
@@ -246,7 +246,7 @@ def predict_proba(self, X=None, train_data=True, return_labels=False):
         for labels, text in data_loader(fn=fn, indices=X,
                                         batch_size=self.batch_size):
             pred = self.clf.predict(text=text, k=len(self.clf.get_labels()))
-            # Get p(s = k | x) matrix of shape (N x K) of pred probs for each x
+            # Get p(labels = k | x) matrix of shape (N x K) of pred probs for each x
             psx = [[p for _, p in sorted(list(zip(*l)), key=lambda x: x[0])] for
                    l in list(zip(*pred))]
             psx_list.append(np.array(psx))
```

Comment:
Do we want to keep fasttext.py and the other `models/` files in the main repo?  Or should they be moved to `cleanlab/examples` or potentially even a new sklearn-wrappers repo?

At the very least, I'd suggest renaming this `models/` folder to something like `example_models/` to make it more clear these models are arbitrary choices and not being utilized by the rest of cleanlab's core functionality.

Reply:
I like the idea of moving to example_models for now. Its a good point. Let's add this to project tracking for 2.0.1+

Reply:
This would be API breaking change though, so we'd need to make it in 2.0.
We can just do it in a followup PR after this one?